Opdracht1.js

```diff
@@ -0,0 +1,203 @@
+            wifi: true,
+            speech: true,
+            hdr: true,
+            bluetooth: true,
+            ambiLight: false,
+        },
+        originalStock: 7,
+        sold: 0,
+    },
+    {
+        type: '43HAK6152',
+        name: 'Ultra HD SMART TV',
+        brand: 'Hitachi',
+        price: 349,
+        availableSizes: [43, 50, 55, 58],
+        refreshRate: 60,
+        screenType: 'LCD',
+        screenQuality: 'Ultra HD/4K',
+        smartTv: true,
+        options: {
+            wifi: true,
+            speech: true,
+            hdr: true,
+            bluetooth: true,
+            ambiLight: false,
+        },
+        originalStock: 5,
+        sold: 5,
+    },
+    {
+        type: '50PUS7304/12',
+        name: 'The One 4K TV',
+        brand: 'Philips',
+        price: 479,
+        availableSizes: [43, 50, 55, 58, 65, 70],
+        refreshRate: 50,
+        screenType: 'LED',
+        screenQuality: 'Ultra HD/4K',
+        smartTv: true,
+        options: {
+            wifi: true,
+            speech: true,
+            hdr: true,
+            bluetooth: true,
+            ambiLight: true,
+        },
+        originalStock: 8,
+        sold: 3,
+    },
+    {
+        type: '55PUS7805',
+        name: '4K LED TV',
+        brand: 'Philips',
+        price: 689,
+        availableSizes: [55],
+        refreshRate: 100,
+        screenType: 'LED',
+        screenQuality: 'Ultra HD/4K',
+        smartTv: true,
+        options: {
+            wifi: true,
+            speech: false,
+            hdr: true,
+            bluetooth: false,
+            ambiLight: true,
+        },
+        originalStock: 6,
+        sold: 3,
+    },
+    {
+        type: 'B2450HD',
+        name: 'LED TV',
+        brand: 'Brandt',
+        price: 109,
+        availableSizes: [24],
+        refreshRate: 60,
+        screenType: 'LED',
+        screenQuality: 'Full HD',
+        smartTv: false,
+        options: {
+            wifi: false,
+            speech: false,
+            hdr: false,
+            bluetooth: false,
+            ambiLight: false,
+        },
+        originalStock: 10,
+        sold: 8,
+    },
+    {
+        type: '32WL1A63DG',
+        name: 'HD TV',
+        brand: 'Toshiba',
+        price: 161,
+        availableSizes: [32],
+        refreshRate: 50,
+        screenType: 'LED',
+        screenQuality: 'Full HD',
+        smartTv: false,
+        options: {
+            wifi: false,
+            speech: false,
+            hdr: true,
+            bluetooth: false,
+            ambiLight: false,
+        },
+        originalStock: 10,
+        sold: 8,
+    },
+];
+
+// Opdracht 1a.
+const tvTypes = inventory.map((tvtype) => {
+    return tvtype.type;
+} )
+console.log(tvTypes);
+
+
+
+// Opdracht 1b.
+inventaris = inventory.originalStock - inventory.sold;
+
+const inStock = inventory.filter((stock) => {
+    if (inventaris > 0 ) {
+        return stock;
+    } else {
+        return false;
+    }
+} )
```

Comment:
Dit had je moeten samenvoegen
```suggestion
const outOfStock = inventory.filter(tv => tv.originalStock === tv.sold);
```